app/controllers/v2/editions_controller.rb

```diff
@@ -0,0 +1,64 @@
+module V2
+  class EditionsController < ApplicationController
+    def index
+      query = Queries::KeysetPagination.new(
+        Queries::GetEditions.new(
+          fields: query_params[:fields],
+          filters: filters,
+        ),
+        **pagination_params
+      )
```

Comment:
Were we swapping out this Queries::GetEditions class? I thought we were going to use a different class as we'd be using a pluck?